src/test/kotlin/sirius/kernel/commons/DatesTest.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.kernel.commons
+
+import java.time.LocalDate
+import java.time.LocalDateTime
+import kotlin.test.*
+
+/**
+ * Tests the [Dates] class.
+ */
+class DatesTest {
+    @Test
+    fun computeLatestDateTime() {
+        val theDayBeforeYesterday = LocalDateTime.now().minusDays(2)
+        val yesterday = LocalDateTime.now().minusDays(1)
+        val now = LocalDateTime.now()
+        assertEquals(now, Dates.computeLatestDate(null, theDayBeforeYesterday, now, yesterday))
+        assertEquals(null, null)
```

Comment:
What are you testing here?

Reply:
With the null, null?
That no excpetion is thrown and that the result is null if I put in null